### DISCUSSION TITLE
Docker use of /run/secrets

### DISCUSSION THREAD

**Discussion by MarcHagen** [directus/directus#2739] [Archive]
In cluster environments its common practice to use secret files instead of environment keys for passwords.

At the moment there is no such implementation for Directus docker, but i would really like to have it so i can use existing secrets to relay information.

**Comment by benhaynes** [comment]
@WoLfulus — any thoughts?

**Comment by WoLfulus** [comment]
Indeed there's no support for secrets right now, and I'm not doing this in the current version of docker images, but once we get the laravel port up and running I'll go through all these docker issues, and this is one in my TODO list for a while now.

**Reply by WoLfulus** [reply]
Laravel version now supports `/var/secrets` + mounting values to different locations by specifying _FILE into the env variable, for example:

`DIRECTUS_DATABASE_HOST_FILE=/var/configs/my_database_host`
